src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -249,20 +249,19 @@ public synchronized void applyDDL(Sequenced<DDLEvent> sequencedEvent) throws Exc
         TableId tableId = TableId.of(project, event.getDatabase(), event.getTable());
         Table table = bigQuery.getTable(tableId);
         // TODO: check schema of table if it exists already
-        // TODO: add a way to get PK of a table through Delta API?
         if (table == null) {
           TableDefinition tableDefinition = StandardTableDefinition.newBuilder()
             .setSchema(Schemas.convert(addSequenceNumber(event.getSchema())))
             .build();
-          String description;
+          String primaryKey;
           if (event.getPrimaryKey().isEmpty()) {
-            description = "Primary Key: _sequence_num";
+            primaryKey = "_sequence_num";
```

Comment:
I don't think this actually works, was probably some placeholder I put here in the spike. We should actually throw a DeltaFailureException saying tables without primary keys are not supported.

Reply:
Sure, will throw exception.